jupyter-book/conditions/perturbation_modeling.ipynb

```diff
@@ -22,12 +22,12 @@
    "metadata": {},
```

Comment:
<p>This could use 1 additional sentence on how the perturbation signature is now used to cluster the data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='130'/>